codegen.yml

```diff
@@ -0,0 +1,40 @@
+overwrite: true
+# schema: "https://raw.githubusercontent.com/GaloyMoney/galoy/main/src/graphql/main/schema.graphql"
+schema: "https://raw.githubusercontent.com/GaloyMoney/galoy/feb351eb62943bef80f8694868a0bf27596b4951/src/graphql/main/schema.graphql"
+documents: 
+  - "app/graphql/*/*.ts"
+generates:
+  app/graphql/generated.ts:
+    plugins:
+      - typescript
+      - typescript-operations
+      - typescript-react-apollo
```

Comment:
@samerbuna fyi, by adding this directive, all the queries/mutations are automatically typed. ie: 

```
import { useCaptchaCreateChallengeMutation } from "@app/graphql/generated"
```

used here https://github.com/GaloyMoney/galoy-mobile/pull/840/files#diff-8b5390c865ce474872401927ca1d6026fb74fc6a313a02f6a5fedea92006e908R25